lib/syskit/models/configured_deployment.rb

```diff
@@ -18,10 +18,14 @@ class ConfiguredDeployment
             # @return [Hash] the name mappings, e.g. the mapping from a task
             #   name in {#model} to the name this task should have while running
             attr_reader :name_mappings
+            # @return [Array] the tasks related to this deployment supposed to be set as
+            #   read only
+            attr_reader :read_only
```

Comment:
This list should really be validated early. Possibly, ideally, at the creation of `ConfiguredDeployment`

That is, `ConfiguredDeployment` should check that all task names in `read_only` are actual names of the deployment.

Otherwise, it will fail at runtime, which is really something we want to avoid as much as possible.